sdks/python/apache_beam/transforms/util_test.py

```diff
@@ -228,6 +228,23 @@ def test_windowed_batches(self):
               7,  # elements in [30, 47)
           ]))
 
+  def test_global_batch_timestamps(self):
+    # Assumes a single bundle, in order...
```

Comment:
Since you erase the values with `len` and `equal_to` is order-agnostic, doesn't this not actually care about order?
```suggestion
    # Assumes a single bundle
```